can_tools/scrapers/official/AZ/az_demographics.py

```diff
@@ -0,0 +1,83 @@
+from tableauscraper import TableauScraper
+import us
+import pandas as pd
+from can_tools.scrapers import variables
+
+from can_tools.scrapers.official.base import StateDashboard
+
+
+class ArizonaVaccineRace(StateDashboard):
+    has_location = False
+    source = "https://www.azdhs.gov/covid19/data/index.php#vaccine-admin"
+    source_name = "ARIZONA DEPARTMENT OF HEALTH SERVICES"
+    location_type = "county"
+    timezone = "US/Mountain"
+
+    state_fips = us.states.lookup("Arizona").fips
+
+    variables = {"total_vaccinations_initiated": variables.INITIATING_VACCINATIONS_ALL}
+    fetch_url = (
+        "https://tableau.azdhs.gov/views/VaccineDashboard/Vaccineadministrationdata"
+    )
+
+    def fetch(self):
+        engine = TableauScraper()
+        engine.loads(self.fetch_url)
+        return engine.getWorksheet(" County using admin county")
+
+    def normalize(self, worksheet: TableauScraper) -> pd.DataFrame:
+
+        # extract data from Tableau Dashboard
+        counties = [
+            t["values"]
+            for t in worksheet.getSelectableItems()
+            if t["column"] == "Admin Address County W/ State Pod"
+        ][0]
+        raw_data = []
+        for county in counties:
+            print("parsing", county)
```

Comment:
Was this just for debugging? Or do we want it checked-in (no preference on my end; just double-checking)?

Reply:
mostly just for debugging to make sure the scraper is working -- I'll remove